curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+Which CSS property is used to set the stack order of an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+position
 
 ---
 
-Placeholder distractor 2
+`bg-green`
 
 ---
 
-Placeholder distractor 3
+`float`
 
 #### --answer--
 
-Placeholder answer
+`z-index`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when you apply the property position: relative to an element in CSS ?
 
 #### --distractors--
 
-Placeholder distractor 1
+It removes the element from document flow
 
 ---
 
-Placeholder distractor 2
+It only moves the element relative to the viewport
 
 ---
 
-Placeholder distractor 3
+This will keep the element in a fixed position relative to the viewport.
 
 #### --answer--
 
-Placeholder answer
+It moves an element relative based on its normal position
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property would you use to fix an element at a certain position on the page so that it does not move when scrolling occurs?
 
 #### --distractors--
 
-Placeholder distractor 1
+`position: absolute`
 
 ---
 
-Placeholder distractor 2
+`position: relative`
 
 ---
 
-Placeholder distractor 3
+`display: block`
 
 #### --answer--
 
-Placeholder answer
+`position: fixed`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+If an element has `position: absolute`, to what is it positioned by default ?
 
 #### --distractors--
 
-Placeholder distractor 1
+The closest ancestor with the attribute `position: fixed`
 
 ---
 
-Placeholder distractor 2
+The closest ancestor that has `position: absolute`
 
 ---
 
-Placeholder distractor 3
+The `<body>` element
 
 #### --answer--
 
-Placeholder answer
+The closest ancestor that has `position: relative`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which positioning method allows an element to stick to a defined position only when you scroll past a certain point?
 
 #### --distractors--
 
-Placeholder distractor 1
+`position: float`
 
 ---
 
-Placeholder distractor 2
+`position: fixed`
 
 ---
 
-Placeholder distractor 3
+`position: absolute`
 
 #### --answer--
 
-Placeholder answer
+`position: sticky`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What would happen if an element had z-index: -1?
 
 #### --distractors--
 
-Placeholder distractor 1
+It will be on top of all the other elements on the page
 
 ---
 
-Placeholder distractor 2
+It will disappear from view.
 
 ---
 
-Placeholder distractor 3
+It will stack on top of all other elements
 
 #### --answer--
 
-Placeholder answer
+It will always be in the back of all the other elements on the page
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How does a floated element affect the layout of surrounding elements?
 
 #### --distractors--
 
-Placeholder distractor 1
+Other elements are placed in relation to the floated element
 
 ---
 
-Placeholder distractor 2
+The floated element is overlapped by surrounding elements that ignore it.
```

Comment:
```suggestion
The surrounding elements ignore the floated element and overlap it.
```

It is a subtle distinction.

The first "elements that ignore it" implies (to me) an explicit state, and the second "surrounding elements ignore" is an implicit state caused by the floated element.

It is definitely possible for elements to overlap the floated element if they are "ignoring it", e.g. are positioned so they can overlap it.